### PR TITLE
feat: add cloud workflow for exporting vector data from Bigtable to Vertex Vector Search

### PR DESCRIPTION
**Pull request by ron-gal**
…x Vector Search

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigtable/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

This is a Bigtable version of https://github.com/cloudspannerecosystem/spanner-ai. Most of the work was copy pasting the Spanner code and adjusting it for Bigtable.

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/vertex-vector-search/tests/system/workflow-test.py [hunk 1]
```diff
@@ -0,0 +1,794 @@
...
+        )
+    )
+
+    # Initialize request argument(s)
+    request = workflows_v1.DeleteWorkflowRequest(
+        name=workflow_full_path,
+    )
+
+    # Make the request
+    operation = workflow_client.delete_workflow(request=request)
```
Comment (daniel-sanche) [inline samples/vertex-vector-search/tests/system/workflow-test.py:99]: Can you put this in a try/finally block? I want to make sure it is being cleaned up properly, even if it exits early

Comment (ron-gal) [reply samples/vertex-vector-search/tests/system/workflow-test.py:99]: Done

File: samples/vertex-vector-search/tests/system/workflow-test.py [hunk 2]
```diff
@@ -0,0 +1,794 @@
...
+    client = bigtable.Client(admin=True, project=project_id)
+    instance = client.instance(instance_id)
+    table = instance.table(table_name)
+    table.create(
+        column_families={
+            "cf": column_family.MaxVersionsGCRule(1),
+        }
+    )
+
+    logger.info("Created {} table on instance {}.".format(table_name, instance_id))
```
[the remark this answers is not in the document]
Comment (ron-gal) [reply samples/vertex-vector-search/tests/system/workflow-test.py:198]: Yes, everywhere this function is called it is followed by cleanup_bigtable_resources. Added try\finally in the one place it was missing, to ensure it happens.

**Comment by daniel-sanche** [comment]
I'm not sure this is the right place for this kind of content. Usually, the `samples/` directory would just contain code snippets surrounded by region tags, that are then pulled into user-facing content on the docsite and/or in the library documentation in the `docs/` folder. I'm not really seeing any region tags in this PR, and it seems more like a full tutorial. Are you sure this is the right place for it? Have you discussed this with the tech writers?

I'd probably suggest moving the contents of this README to a [page on the docsite](https://cloud.google.com/bigtable/docs/overview), pulling in some snippets [like this one](https://github.com/googleapis/python-bigtable/blob/ded8fb74425becbd1dad2170a251988b5f05697a/samples/vertex-vector-search/tests/system/workflow-test.py#L581) to show how to interact with it using Python, and linking to that page in the README here. But let me know if you've already considered that

**Comment by ron-gal** [comment]
Merged in https://github.com/GoogleCloudPlatform/workflows-demos/pull/114

**Pull request closed without merging**